src/main/java/com/exasol/adapter/dialects/elasticsearch/ElasticSearchMetadataReader.java

```diff
@@ -0,0 +1,41 @@
+package com.exasol.adapter.dialects.elasticsearch;
+
+import java.sql.Connection;
+
+import com.exasol.adapter.AdapterProperties;
+import com.exasol.adapter.dialects.*;
+import com.exasol.adapter.jdbc.*;
+
+/**
+ * Metadata reader implementation for reading metadata from an ElasticSearch remote database.
+ */
+public class ElasticSearchMetadataReader extends AbstractRemoteMetadataReader {
+
+    /**
+     * Creates a new instance of {@link ElasticSearchMetadataReader}.
```

Comment:
```suggestion
     * Create a new instance of {@link ElasticSearchMetadataReader}.
```

Reply:
We use Create Build Get (without s)  to have the same style for all javadocs

Reply:
The purpose of the javadoc in a method is to reply the question "What this method does?", which is always in present simple, as you can also see it as the standard way across the whole open jdk.

You could argue that one could write it in the infinitive form, as "To create" or "Creating", but just "Create" is not proper english.